scripts/test-ci.sh

```diff
@@ -7,11 +7,10 @@ npm test
 
 # we need to build to run functional tests
 NODE_ENV=production npm run build
+rm -rf functional-tests/screenshots
```

Comment:
The folder is already automatically created by `wdio-visual-regression-service`.